jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/AdvanceRuntimeDataServiceImpl.java

```diff
@@ -53,7 +53,7 @@ public AdvanceRuntimeDataServiceImpl() {
     public List<ProcessInstanceWithVarsDesc> queryProcessByVariablesAndTask(List<QueryParam> attributes,
                                                                             List<QueryParam> processVariables,
                                                                             List<QueryParam> taskVariables,
-                                                                            List<String> owners,
+                                                                            QueryParam owners,
```

Comment:
signatures of this one should not be changed as you are removing api that is already shipped within the product.
Create a new signature for this one (override)

the old one will call the new one with the QueryParam.ALL

With that you won't need to change any test. Just add a new one with ANY

Reply:
done